pkg/integration/components/test.go

```diff
@@ -41,7 +40,7 @@ type NewIntegrationTestArgs struct {
 	// takes a config and mutates. The mutated context will end up being passed to the gui
 	SetupConfig func(config *config.AppConfig)
 	// runs the test
-	Run func(shell *Shell, input *Input, assert *Assert, keys config.KeybindingConfig)
+	Run func(t *TestDriver, keys config.KeybindingConfig)
```

Comment:
not generally a fan of single-letter variable names, but we already have the precedence of 't' in typical go tests, and other frameworks do a similar thing (e.g. `cy` for Cypress)